base/reflection.jl

```diff
@@ -153,6 +153,8 @@ methods(f::ANY,t::ANY) = methods(call, tt_cons(isa(f,Type) ? Type{f} : typeof(f)
 function _methods(f::ANY,t::ANY,lim)
     if isa(t,Type)
         _methods(f, Any[t.parameters...], length(t.parameters), lim, [])
+    elseif isa(t,TypeVar)
```

Comment:
Looks like it can be deleted without consequence. I suspect this was a desperation move on my part to get stuff to build, from before I discovered the correct `inference.jl` incantations. Will delete.